src/main/resources/db/migration/V6.0.0.5__9686-move-harvestingclient-id.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE dvobject ADD COLUMN IF NOT EXISTS harvestingclient_id BIGINT;
+
+update dvobject dvo set harvestingclient_id = s.harvestingclient_id from
+(select id, harvestingclient_id from dataset d where d.harvestingclient_id is not null) s
+where s.id = dvo.id; 
+
+--ALTER TABLE dataset drop COLUMN IF EXISTS harvestingclient_id;
```

Comment:
the query looks fine and we will have fully tested (and have backups); this can be uncommented back in